packages/chat-client/src/client.ts

```diff
@@ -213,6 +220,11 @@ export class ChatClient extends IChatClient {
     }
   };
 
+  public addContact: IChatClient["addContact"] = ({ account, publicKey }) => {
+    this.chatContacts.set(account, { accountId: account, publicKey });
+    return Promise.resolve();
```

Comment:
Needed? I think it can be non-async if not actually needed. The spec just makes an assumption that this will be async, but it also doesn't apply in other places where the spec indicates a `Promise` return.